testing/GaussFermi_float128.py

```diff
@@ -16,5 +15,3 @@
 devsim.set_parameter(name = "extended_model", value=True)
 devsim.set_parameter(name = "extended_equation", value=True)
 
-import GaussFermi
```

Comment:
This import loads and runs `GaussFermi.py` as a script after setting extended precision.

Reply:
fixed!